src/System.Runtime.Serialization.Primitives/src/System/Runtime/Serialization/FormatterServices.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+namespace System.Runtime.Serialization
+{
+    public static class FormatterServices
+    {
+    	public static object GetUninitializedObject(Type type)
+    	{
+    		// On .NET Native, this is not implemented
+    		throw new NotImplementedException();
+    	}
+    }
+}
```

Comment:
Actually we had it working with a new contract version.  The downside is that the API is only exposed for net46 or later.